Session2_Intro_To_Machine_Learning_For_Meteorological_Science/requirements.yml

```diff
@@ -0,0 +1,16 @@
+name: spaceapps-mo-ml-intro
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - jupyterlab>=3.4.7
+  - pandas>=1.5.0
+  - numpy>=1.23.3
+  - matplotlib>=3.6.0
+  - scipy>=1.9.1
+  - ipykernel>=6.16.0
+  - ipython>=8.5.0
+  - tensorflow>=2.0
+  - scipy>=1.9.1
+  - sklearn>=1.0.2
```

Comment:
The package name for sklearn on conda is 'scikit-learn', the environment will resolve when the name is updated